.github/workflows/build-release.yaml

```
@@ -0,0 +1,58 @@
+name: Test, create and publish a Docker image
+
+on: [push]
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  build-and-push-image:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+      
+      - name: Build jar 
+        run: mvn compile && mvn package
+
+      - name: Log in to the Container registry
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      
+      - name: install xmllint
+        run: sudo apt  install libxml2-utils
```

Do we need to `-y` this?
```suggestion
        run: sudo apt-get install -y libxml2-utils
```

---

It seems to work without:
https://github.com/distributed-technologies/kafka-http-sink-connector/pull/1/checks

---

interesting